ably.d.ts

```diff
@@ -700,6 +700,13 @@ declare namespace Types {
     quickAck?: boolean;
   };
 
+  type MessageFilter = {
+    name?: string;
+    refId?: string;
+    refType?: string;
+    hasRef?: boolean;
+  };
+
   class RealtimeChannelCallbacks extends RealtimeChannelBase {
```

Comment:
We also need to update the `RealtimeChannel.unsubscribe` signatures.

Reply:
RealtimeChannelCallbacks doesn't have an unsubscribe signature, and RealtimeChannelBase has been updated